docs/gates.md

```diff
@@ -20,87 +20,109 @@ nav_order: 3
 ## Introduction
 
 
-Logic gates are the basic building blocks of any digital system. 
-It is an electronic circuit having one or more than one input and only one output. The relationship between the input and the output is based on a certain logic. Based on this, logic gates are named as AND gate, OR gate, NOT gate etc.
+It is a device which performs logic operations on one or more logical inputs, and produces a single logical output.
+Logic gates can be categorized into 3 groups:
+1. Basic Gates: NOT, AND, OR
+2. Universal Gates: NAND, NOR
+3. Arithmetic Gates: X-OR, X-NOR
+
+<h2>Truth Table</h2>
+
+The Table which contains all logical possibilities is known as truth table.
 
 ## NOT gate
 
-The NOT gate is also known as an inverter because the output is the exact opposite of the input. It has one input and one output. The two possibilities are written out in the table below. Tables listing all logical possibilities like this are known as truth tables.
+The NOT gate is also known as an inverter because it inverts the logic sense of a binary variable. It has one input and one output. The Truth table for NOT gate is given below
```

Comment:
it produces the exact opposite of the input as output.

The current statement appears complex.

Reply:
Done